tests/lib/cli-engine/formatters/junit.js

```diff
@@ -12,12 +12,37 @@
 //------------------------------------------------------------------------------
 
 const assert = require("chai").assert,
-    formatter = require("../../../../lib/cli-engine/formatters/junit");
+    formatter = require("../../../../lib/cli-engine/formatters/junit"),
+    process = require("process");
 
 //------------------------------------------------------------------------------
 // Tests
 //------------------------------------------------------------------------------
 
+/**
+ * Returns the suppliedFilePath for each test
+ * @returns {string} the filepath in a format that would be expected on either a win32 system or POSIX system
+ * @private
+ */
+function suppliedFilePath() {
```

Comment:
Thoughts on making these two actual constants instead of having to call the functions everywhere?

Reply:
Fine for me, changed